com.bitwig.BitwigStudio.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.bitwig.BitwigStudio</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>BitwigStudio</name>
+  <summary>BitwigStudio digital audio workstation</summary>
+  <description>
+    <p>
+      BitwigStudio is an application to produce music. As a digital
+      audio workstation (DAW) it supports VST and VST3 audio plugins.
```

Comment:
On the website we have:
Modern music production for Windows, macOS, and Linux.

Maybe we could use the same for now?

Reply:
Since the metadata is CC0 I didn't feel like copying the website.

Reply:
Go ahead and copy it ;-)